src/bash_history.rs

```diff
@@ -7,12 +7,13 @@ use std::fs::File;
 use std::io::Read;
 
 fn read_ignoring_utf_errors(path: &PathBuf) -> String {
-    let mut f = File::open(path).expect(format!("McFly error: {:?} file not found", &path).as_str());
+    let mut f = File::open(path).unwrap_or_else(|_| panic!("McFly error: {:?} file not found", &path));
```

Comment:
All the switched to `unwrap_or_else` are good-- they'll be more performant, thanks.